xcode/Subconscious/Shared/Components/Detail/UserProfileDetailMetaSheet.swift

```diff
@@ -197,7 +197,7 @@ struct UserProfileDetailMetaSheet: View {
             }
         }
         .presentationDragIndicator(.hidden)
-        .presentationDetents([.medium, .large])
+        .presentationDetents([.medium])
```

Comment:
No reason to fill the screen given how limited the options are in the Profile Metasheet